### DISCUSSION TITLE
Add DNSCRYPT

### DISCUSSION THREAD

**Discussion by Genddos** [pivpn/pivpn#1753] [Feature requests]
please add dnscrypt to your pivpn package.  Git has something similar.

**Comment by coolapso** [comment]
I Believe you might be mixing up some stuff, PiVPN is not a package, PiVPN is an installer. and I believe what you are requesting out of pivpn scope. 

DNS Requests are handled by the DNS Servers you setup. 

First the DNS providers you use must support it, second it requires installation of a dnscrypt client such as dnscrypt-proxy and has been decided that pivpn does not install anything else other than the protocols and its own dependencies. 

If you wish to use DNSCRypt together with OpenVPN or Wireguard you should set it up separately. If you do so and wish to document it, ill be glad to accept an how to regarding this topic on pivpn documentation

**Comment by Genddos** [comment]
I apologize for the translation difficulties. I meant to add your DNSCrypt installation script to the script. Implementation example :
https://github.com/zzzkeil/Wireguard-DNScrypt-VPN-Server/tree/master
